control-plane/pkg/reconciler/broker/broker.go

```diff
@@ -234,28 +258,64 @@ func incrementVolumeGeneration(generation uint64) uint64 {
 	return (generation + 1) % (math.MaxUint64 - 1)
 }
 
-func (r *Reconciler) topicDetailFromBrokerConfig(broker *eventing.Broker) *sarama.TopicDetail {
-	// TODO use broker configurations - see https://github.com/knative-sandbox/eventing-kafka-broker/issues/9
+func (r *Reconciler) resolveBrokerConfig(logger *zap.Logger, broker *eventing.Broker) (Config, error) {
```

Comment:
I think this function in case a "specific" broker config exists, it should directly return the merged config between the default one and the specific one, so you avoid scattering the code to merge the 2 configs around

Reply:
Sure.